scripts/setup/nrfconnect/get_zap.py

```diff
@@ -198,4 +204,5 @@ def main():
 
 
 if __name__ == '__main__':
+
```

Comment:
nit but why are blank lines added in `if` or a method block but only sometimes. Let's be consistent :)